Make_Tables/mysqlconnect.py

```diff
@@ -5,9 +5,9 @@
 directory = os.getcwd()
 envindex = directory.find('Make_Tables')
 if envindex!=-1:  #If current working directory is Make_Tables
-	load_dotenv(directory[:envindex]+"/.env")
+	load_dotenv(directory[:envindex]+"/.sampleenv")
```

Comment:
- [ ] read from .env only, .sampleenv is so that people know what their .env requires.
(This is so that people don't have to worry about git tracking changes to .env also add a step to readme about creating .env using .sampleenv)